src/objects/zcl_abapgit_object_fugr.clas.abap

```diff
@@ -1179,8 +1181,14 @@ CLASS zcl_abapgit_object_fugr IMPLEMENTATION.
     lv_program_name = main_name( ).
     ls_progdir = read_progdir( lv_program_name ).
 
-    serialize_texts( iv_prog_name = lv_program_name
-                     ii_xml       = io_xml ).
+    IF io_xml->i18n_params( )-translation_languages IS INITIAL.
+      " Old I18N option
+      serialize_texts( iv_prog_name = lv_program_name
+                       ii_xml       = io_xml ).
+    ELSE.
+      " New LXE option
+      serialize_translations( io_xml ).
```

Comment:
`serialize_texts` and `serialize_translations` - the names looks confusing to me, it is not clear which is which. I'd suggest `serialize_lxe_texts` or `serialize_texts_lxe`

Reply:
done